### DISCUSSION TITLE
26.1

### DISCUSSION THREAD

**Discussion by thdenton** [grahampugh/erase-install#264] [Q&A] [answered]
Hello

Just been testing out the new version of your tool. I get UNDERGOING TEST shortly after the download stage has completed.  Is this normal?

Also once preparing install has completed. I noticed it quite out off full depnotify window. Before restating. Is this normal?  Haven't noticed it before on the old version.

**Comment by grahampugh** [comment, marked as answer]
Please download the package again, the UNDERGOING TEST message was a test message that slipped out into production. I've removed it now. 

As for DEPNotify quitting, I did not change anything there, so not sure why you would see any difference.
